api/tests/core/logging/tests.py

```diff
@@ -128,7 +127,7 @@ def test_serialization(self):
                 "user": user,
                 "bytes": b"encod\xc3\xa9",
                 "nested_complex_object": [{"foo": ["bar"]}],
-                "event": amplitude.BaseEvent(event_type="event type"),
+                "event": {"event_type": "event type"},
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ça disparait pas complètement ça ?‏

Reply:
bien vu 👍